templates/web/base/admin/reports/index.html

```diff
@@ -14,6 +14,7 @@
         <th>[% loc('Name') %]</th>
         <th>[% loc('Body') %]</th>
         <th>[% loc('State') %]</th>
+        [% IF c.user.has_permission_to('assign_report_to_user', c.cobrand.body.id) %]<th>[% loc('Assigned to') %]</th>[% END %]
```

Comment:
Hmm, is this a big enough feature that it gets its own column in what is already quite a wide table?
Would it be okay as an extra row in the "Body" column perhaps, given it's associated with the body?
Not sure.

Also the permission here - I think the usual way we do this is `c.user.has_body_permission_to(permission)` which does a `user.from_body` check.